industrial_ci/ci_main.sh

```diff
@@ -82,16 +93,18 @@ BUILDER=catkin
 ROSWS=wstool
 # For compatibilility with hydro catkin, which has no --verbose flag
 CATKIN_TEST_RESULTS_CMD="catkin_test_results"
-if [ catkin_test_results --verbose 1>/dev/null 2>/dev/null; then CATKIN_TEST_RESULTS_CMD="catkin_test_results --verbose"; fi
```

Comment:
IMHO catkin_test_results should always be run in verbose mode. It is the main result of the CI service.